### PR TITLE
Simplify install instruction in README

### PR DESCRIPTION
**Pull request by vuillaut**
The installation instructions are now in line with the installation done by Travis (and are much simpler).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,19 +23,20 @@ master branch status: [![Build Status](https://travis-ci.org/cta-observatory/cta
...
+PROTOZFITS_VERSION=v1.4.2
+CTAPIPE_IO_LST_VERSION=master
+pip install https://github.com/cta-observatory/ctapipe/archive/$CTAPIPE_VERSION.tar.gz
+pip install https://github.com/cta-sst-1m/protozfitsreader/archive/$PROTOZFITS_VERSION.tar.gz
+pip install https://github.com/cta-observatory/ctapipe_io_lst/archive/$CTAPIPE_IO_LST_VERSION.tar.gz
+```
+
+If you made local changes to `lstchain` that you want to install, run
+```
 python setup.py install
```
Comment (dneise) [inline README.md]: I would recommend `pip installe -e .` so that local changes take effect.

File: README.md [hunk 2]
```diff
@@ -22,19 +23,20 @@ master branch status: [![Build Status](https://travis-ci.org/cta-observatory/cta
...
 cd cta-lstchain
+conda env create --name cta --file environment.yml
+conda activate cta
+CTAPIPE_VERSION=master
+PROTOZFITS_VERSION=v1.4.2
+CTAPIPE_IO_LST_VERSION=master
+pip install https://github.com/cta-observatory/ctapipe/archive/$CTAPIPE_VERSION.tar.gz
+pip install https://github.com/cta-sst-1m/protozfitsreader/archive/$PROTOZFITS_VERSION.tar.gz
+pip install https://github.com/cta-observatory/ctapipe_io_lst/archive/$CTAPIPE_IO_LST_VERSION.tar.gz
+```
```
Comment (dneise) [inline README.md]: I find this env-var definition cumbersome, why not resolve them?

**Comment by rlopezcoto** [comment]
Hi, it looks good to me, after @dneise's comments have been taken into account, I'll merge this

**Comment by vuillaut** [comment]
Sorry I forgot this PR was pending.
`pip install -e .` is possible if people clone the repo.
Here they can just copy paste and be ready to go.
But I don't know what's simpler or better.

**Comment by rlopezcoto** [comment]
I also find easier that the users copy and paste the text

**Comment by dneise** [comment]
> pip install -e . is possible if people clone each repo.
Here they can just copy paste and be ready to go.
But I don't know what's simpler or better.

I do not understand this. I just tested the following:

 - make a new empty fresh conda environment (with py37) - this fakes a new user, with no idea of python who just installed anaconda.

 - then did exactly this:
```
pip install https://github.com/cta-observatory/ctapipe/archive/master.tar.gz
pip install https://github.com/cta-sst-1m/protozfitsreader/archive/v1.4.2.tar.gz
pip install https://github.com/cta-observatory/ctapipe_io_lst/archive/master.tar.gz
git clone https://github.com/cta-observatory/cta-lstchain.git
pip install -e cta-lstchain
```

The advantage of `pip install -e` over `python setup.py install` is simply: If they modify some code in lstchain, then it immediately takes effect and they do need to reinstall. Which in my experience is less surprising to new users, since this is exactly what they are used to, when they just did their first Python tutorials involving a single python file, they are writing some functions into and then importing that file in an ipython shell and trying it out.... 

Not sure if such beginners are your intended audience though... for more advanced pythonistas this should however not pose a problem.

**Comment by vuillaut** [comment]
Hi

Sorry I thought you meant `pip install -e .` for each package.
My bad, wrong interpretation.

In this case I agree with you, you can make the change to use pip and merge.
Sorry I don't have a computer to do it.

Le jeu. 4 avr. 2019 à 15:40, Dominik Neise <notifications@github.com> a
écrit :

**Pull request merged**